### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ckim328**
This is an example PR for having issue templates. This is subject to change, so no need to approve/merge soon

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (Shabirmean) [inline .github/ISSUE_TEMPLATE/feature-request.md:1]: Why do we have two files for feature requests?

Comment (ckim328) [reply .github/ISSUE_TEMPLATE/feature-request.md:1]: my bad! I believe it was autogenereated, ill delete! good catch! 👍

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (NimJay) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Nitpick: Should we use `kebab-case` for folder and file names (to be consistent with most of the repo)? 
Sorry, this is mostly a "me" problem. 😁 
So `ISSUE_TEMPLATE/bug_report.md` would be `issue-template/bug-report.md`.

I do realize that the existing pull-request template uses `snake_case`, but we can change that to `kebab-case` later:
https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/master/.github/pull_request_template.md

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+**Logs**
+Add logs to help explain your problem
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Specs (please complete the following information):**
+ - OS: [e.g. MacOS Big Sur v11.6]
+ - Versions [e.g. Docker v20.10.8]
```
Comment (NimJay) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Suggestion: `Versions:` with a colon?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
```
Comment (NimJay) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: Question: I am wondering if we should (and if we can) add `anthos-dpe` as an assignee to all 3 of these templates?

**Review by NimJay — COMMENTED** [review]
@ckim328 Thanks again for initiating this much needed change! Things are looking good! 🙌 I've just left a few comments.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (ckim328) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: hmm i was initially thinking that, but i was also weary of the potential notification overload 😅  we could just add it for the time being, and see how it works?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+2. Ran command '....'
+3. See error
+
+**Logs**
+Add logs to help explain your problem
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Specs (please complete the following information):**
```
Comment (NimJay) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Thought: I am happy to remove the "(please complete ... information)" message. I'm guessing you want to indicate that this section is **required**, but then the other sections seem like they're optional. 😅

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (NimJay) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: Praise: I'm digging the simplicity of these headings!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (NimJay) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: > potential notification overload

Ah, trueee! You're right. Plus we always have someone on rotation monitoring the dashboard. 
Let's not add assignees.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 5 above)
Comment (ckim328) [reply .github/ISSUE_TEMPLATE/bug_report.md:25]: sounds good, i can take it out! 😄

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (NimJay) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Sweet! Thanks, Christine.
I just realized that the all uppercase `ISSUE_TEMPLATE` is a [GitHub thing](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms).

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+### Logs  
+<!-- Add logs to help explain your problem -->
+
+### Screenshots 
+<!-- If applicable, add screenshots to help explain your problem -->
+
+### Environment 
+<!--  - OS: [e.g. MacOS Big Sur v11.6] -->
+<!--  - Kubernetes distribution, version,: [e.g. minikube, GKE (Standard or Autopilot), EKS, AWS ... ] -->
```
Comment (NimJay) [inline .github/ISSUE_TEMPLATE/bug-report.md:27]: `version,:` -> `version:`?

**Review by NimJay — APPROVED** [review]
I left one additional comment/nitpick (about `version,:`), but other than, everything looks good!
Thanks again for doing this, @ckim328!
Approved.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 7 above)
Comment (ckim328) [reply .github/ISSUE_TEMPLATE/bug-report.md]: lol thanks for catching that!

**Pull request merged**